blog/.vuepress/config.js

```diff
@@ -9,6 +9,23 @@ module.exports = {
     logo: "/vue-logo-website-trimmed-transparent.png",
     lastUpdated: "Last Updated",
     nav: [
+      {
+        text: "Events",
+        items: [
+          {
+            text: "Next Event",
+            link: "/events/index.html"
+          },
+          {
+            text: "Tags",
+            link: "/tags/index.html"
+          },
+          {
+            text: "Archives",
+            link: ""
```

Comment:
I would add this comment, for our future self:
```suggestion
            link: "" // Defined at compile time in blog/.vuepress/plugins/sidebar/create.js
```